src/components/pages/LogIn.tsx

```diff
@@ -238,10 +238,16 @@ class ConfirmationCodeForm extends React.Component<
   };
 
   componentDidMount() {
+    this.setState({
+      timerInterval: setInterval(this._calculateTimeLeft, 1000)
```

Comment:
see https://discuss.reactjs.org/t/setinterval-in-state/7955, `setState` usually triggers a render which isn't needed here.

Also reminder about line 451 `new Date()`